### PR TITLE
New Rule: Fake Message delivery failure

### PR DESCRIPTION
**Pull request by morriscode**

### CODE REVIEW

**Comment by morriscode** [comment]
Changing to draft. Plan to revert line 4 to 
`and length(headers.references) == 0`

File: detection-rules/impersonation_message_delivery_failure.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: "Fake message delivery failure "
+description: |
+  This rule detects inbound messages that appear to be from a mail delivery failure system, but lack thread references and do not contain any RFC822 attachments.
```
Comment (jkamdjou) [inline detection-rules/impersonation_message_delivery_failure.yml:3]:
```suggestion
  Detects inbound messages that appear to be from a mail delivery failure system, but lack thread references and do not contain any RFC822 attachments.
```

File: detection-rules/impersonation_message_delivery_failure.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: "Fake message delivery failure "
```
Comment (jkamdjou) [inline detection-rules/impersonation_message_delivery_failure.yml:1]:
```suggestion
name: "Fake message delivery failure"
```

**Comment by morriscode** [comment]
Interesting volume on this, I'd love to see some of these samples.

**Comment by morriscode** [comment]
New revisions look promising much lower volume. More in line with what I'd expect to see.

**Comment by morriscode** [comment]
https://www.notion.so/sublimesecurity/Re-work-fake-message-deliver-failure-aefc33ffdebc4a19841003ddc6796dec?pvs=4

**Pull request closed without merging**